sumatra/core.py

```diff
@@ -87,16 +87,36 @@ def _get_process_children(pid):
     return [int(child_pid) for child_pid in stdout.split()]
 
 
-class Registry(object):
+class _SingletonType(type):
```

Comment:
Agreed. My intention was to not have that in the public api, but it's true that this is not correct.